### PR TITLE
feat: Add HA statistics metrics and Go module

### LINKED ISSUE #19: Include metrics for HA member
As HA members, at least in our case have no own IP address, some stats for the HA peers should be also included.    

Initially this is what I would need: 
- `fortigate_ha_member_info{hostname="fw01", serial="FGXXXXXXX", vclusterid="0"} 1` 
   path: /monitor/system/ha-peer

later there could be performance metrics like
- `fortigate_ha_member_sessions{hostname="fw01"} 100`
- `fortigate_ha_member_packets_total{hostname="fw01"} 100000`
- `fortigate_ha_member_virus_events_total{hostname="fw01"} 100`
- `fortigate_ha_member_network_usage_ratio{hostname="fw01"} 0.1`
- `fortigate_ha_member_bytes_total{hostname="fw01"} 1000000`
- `fortigate_ha_member_ips_events_total{hostname="fw01"} 100`
- `fortigate_ha_member_cpu_usage_ratio{hostname="fw01"} 0.1`
- `fortigate_ha_member_memory_usage_ratio{hostname="fw01"} 0.1`
path: /monitor/system/ha-statistics

I'm interested in implementing this.

### PR DESCRIPTION
**Pull request by secustor**
Closes #19 

Adds metrics for HA members of firewalls and a GO module definition to enable development outside of GOPATH and fixing the dependencies. 

I have dropped the vclusterid label as for this I would added a own API call and I don't need it atm anyway

### CODE REVIEW

**Comment by secustor** [comment]
@bluecmd Please review

**Comment by bluecmd** [comment]
Sorry, GitHub doesn't notify me unless somebody pings me. This seems to be a common issue on GitHub. I'll get right to it!

File: testdata/ha-statistics.jsonnet [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
```
Comment (bluecmd) [inline testdata/ha-statistics.jsonnet:1]: Please add a comment what API path this is gotten from so I can update it for e.g. 6.4 in the future

File: probe.go [hunk 2]
```diff
@@ -577,8 +577,131 @@ func probe(ctx context.Context, target string, registry *prometheus.Registry, hc
 			probeFirewallPolicies(c, registry) &&
 			probeInterfaces(c, registry) &&
 			probeVPNStatistics(c, registry) &&
-			probeIPSec(c, registry)
+			probeIPSec(c, registry) &&
+			probeHaStatistics(c, registry)
```
Comment (bluecmd) [inline probe.go:581]: Go community prefers acronyms to be capitalized. s/Ha/HA everywhere.

**Review by bluecmd — CHANGES_REQUESTED** [review]
Looks very good, thanks! Minor changes requested.

**Review by bluecmd — APPROVED** [review]

**Pull request merged**